Sources/NIOTransportServices/NIOTSNetworkEvents.swift

```diff
@@ -43,6 +43,19 @@ public enum NIOTSNetworkEvents {
         /// Create a new ``NIOTSNetworkEvents/BetterPathUnavailable`` event.
         public init(){ }
     }
+    
+    /// ``ViabilityUpdate`` is triggered when the OS informs NIO that communication
+    /// with the remote endpoint is possible, indicating that the connection is viable.
+    public struct ViabilityUpdate: NIOTSNetworkEvent {
+
+        /// The current viability for the connection
+        public let isViable: Bool
```

Comment:
Minor nit, this should probably be var.

```suggestion
        public var isViable: Bool
```